Use guidm cursor fns (#1402)
DFHack/scripts: changelog.txt

- `devel/export-map`: export map tile data to a JSON file
- `autocheese`: automatically make cheese using barrels that have accumulated sufficient milk
- `gui/spectate`: interactive UI for configuring `spectate`

> - `launch`: (reinstated) thrash your enemies with a flying suplex

Reviewer: ```suggestion
- `launch`: (reinstated) new adventurer fighting move: thrash your enemies with a flying suplex
```

> - `putontable`: (reinstated) make an item appear on a table like in adventure mode

Reviewer: ```suggestion
- `putontable`: (reinstated) make an item appear on a table
- `devel/query`: support adventure mode
- `devel/tree-info`: support adventure mode
- `hfs-pit`: support adventure mode
- `colonies`: support adventure mode
- `toggle-kbd-cursor`: support adventure mode
```

## New Features
- `advtools`: new ``advtools.fastcombat`` overlay (enabled by default) allows you to skip combat animations and the announcement "More" button by mashing the movement keys

## Fixes
- `position`: support for adv mode look cursor

> - `devel/query`, `devel/tree-info`, `hfs-pit`, `colonies`, `toggle-kbd-cursor`: now function in adventure mode

Reviewer: ```suggestion
```

> - `hfs-pit`: fix up tiletypes of pit walls, better placement of stairs (w/r/t eerie pits and ramp tops)

Reviewer: ```suggestion
- `hfs-pit`: use correct wall types when making pits with walls
```

> - `modtools/create-item`: ``hackWish`` now respects ``opts.pos`` and will spawn items there if provided

Reviewer: ```suggestion
```

## Misc Improvements
- `hide-tutorials`: handle tutorial popups for adventure mode
- `hide-tutorials`: new ``reset`` command that will re-enable popups in the current game (in case you hid them all and now want them back)
- `gui/notify`: moody dwarf notification turns red when they can't reach workshop or items
- `gui/confirm`: in the delete manager order confirmation dialog, show a description of which order you have selected to delete
- `position`: display both adventurer and site pos simultaneously. Display map block pos+offset of selected tile.

> - `gui/create-item`: now accepts a ``pos`` argument of where to spawn items

Reviewer: ```suggestion
- `gui/create-item`: now accepts a ``pos`` argument of where to spawn items
- `modtools/create-item`: exported ``hackWish`` function now supports ``opts.pos`` for determining spawn location
- `hfs-pit`: improve placement of stairs w/r/t eerie pits and ramp tops
```
